docs/product-spec.md

```diff
@@ -21,52 +21,52 @@ By offering simple, readily available, off-the-shelf hardware and
 robust software, we can enable 3 election staffers to manage mailing
 and receiving 50,000 ballots.
 
-
 ## VxMail
 
 We propose VxMail, a set of tools to help small jurisdictions quickly scale
 up their vote-by-mail capacity.
 
-### Components
+VxMail includes the following components:
 
-VxMail includes:
-* a Ballot Manager to create and proof mail ballots, schedule printing
-  and mailing, log receipt of returned ballots, and verify voter
+- **VxMail Ballot Manager** is used to create and proof ballots, schedule
+  printing and mailing, log receipt of returned ballots, and verify voter
   signatures.
-* tools to efficiently open envelopes and extract ballots.
-* a Ballot Tabulator to scan and tabulate ballots.
+- hardware to efficiently open envelopes and extract ballots.
+- **VxMail Ballot Tabulator** is used to scan and tabulate ballots.
 
-The Ballot Manager works online for connecting with print-and-mail
+VxMail Ballot Manager works online for connecting with print-and-mail
 services, and potentially to the state's Election Management
-System. The Ballot Tabulator is offline for security purposes.
+System. VxMail Ballot Tabulator is offline for security purposes.
+
+The step-by-step process of using VxMail is as follows (with possible adaptations for the laws of local jurisdictions)…
 
-### Printing and mailing ballots
+### Proofing, Printing, and Mailing Ballots
 
 The VxMail Ballot Manager lets an election official:
 
-* upload the election definition
-* proof generated ballots on screen
-* upload the voter mailing list (name, mailing address, voter ID, ballot style)
-* order those ballots printed and mailed, with a 3-day mailing turnaround.
-* track ballot delivery to voters
-* track ballot return from voters
+- upload the election definition
+- proof generated ballots on screen
+- upload the voter mailing list (name, mailing address, voter ID, ballot style)
+- order those ballots printed and mailed, with a 3-day mailing turnaround.
+- track ballot delivery to voters
+- track ballot return from voters
 
-All ballot printing, stuffing, and mailing is managed by VotingWorks
-and its partners, while election officials can track that process via
-the Ballot Manager. Voters receives a ballot and return envelope,
+All ballot printing and mailing is managed by VotingWorks
```

Comment:
why remove stuffing? It's an important part of the process, I wouldn't want election officials to think they have to stuff the envelopes.
```suggestion
All ballot printing, stuffing, and mailing is managed by VotingWorks
```

Reply:
Sure. Let's be explicit. 

My though: If we're dexcribing that a mail house is printing and mailing, it seems to be clear that they are also doing the stuffing. 

Also thought it might be wise to avoid the term "stuffing" as in "stuffing ballot boxes".

Reply:
Another reason I had removed "stuffing" is that there are other task details not explicitly mentioned such as mail merge of voter addresses for mailing.